src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -16,7 +14,9 @@ class SettingsLayout extends React.Component {
     router: PropTypes.object,
     routes: PropTypes.array,
   };
-
+  state = {
+    navVisible: true,
+  };
```

Comment:
nit: would be nice to have space between this statment, since that matches what we typically do